bayesian/api/api_v2.py

```diff
@@ -68,7 +68,8 @@ def get_token():
     try:
         # return default key
         THREESCALE_PREMIUM_USER_KEY = os.getenv('THREESCALE_PREMIUM_USER_KEY')
-        return jsonify({"key": THREESCALE_PREMIUM_USER_KEY}), 200
+        PREMIUM_URL = os.getenv('PREMIUM_URL')
+        return jsonify({"url": PREMIUM_URL, "key": THREESCALE_PREMIUM_USER_KEY}), 200
```

Comment:
corresponding changes in the unit tests should be done as well.

Reply:
Yes.On it.

Reply:
I feel there is no modicfication required in the tests. The response code test is already done.

Reply:
add the new key "url" in the test

Reply:
Done